src/librustdoc/html/static/storage.js

```diff
@@ -88,7 +88,7 @@ function getCurrentValue(name) {
     return null;
 }
 
-function switchTheme(styleElem, mainStyleElem, newTheme) {
+function switchTheme(styleElem, mainStyleElem, newTheme, dontSaveNewValue) {
```

Comment:
I'm a bit wary of naming the parameter "don't save new value", just because it feels weird to name a boolean parameter after the opposite of what it does. Maybe something like `initialLoad` or `skipStorage`? Or even inverting the value that is sent on the initial call and calling it `persist`?

Reply:
I like `skipStorage`! Let's go for it! :)